<launch>
  <include file="$(find ir_sensor)/launch/IR_sensor.launch" />

  <!-- Publish  -->
  <node name="ir_safety_controller" pkg="ir_sensor" type="ir_safety_controller_2.py" output="screen">
    <remap from="ir_cmd_vel" to="cmd_vel_mux/input/ir_safety_controller"/>
  </node>

  <!-- nodelet manager -->
  <!-- <node pkg="nodelet" type="nodelet" name="mobile_base_nodelet_manager" args="manager"/> -->
  <!-- <node pkg="nodelet" type="nodelet" name="cmd_vel_mux" -->
  <!--       args="load yocs_cmd_vel_mux/CmdVelMuxNodelet mobile_base_nodelet_manager"> -->
  <!--   <param name="yaml_cfg_file" value="$(find ir_sensor)/param/mux_plus.yaml" /> -->
  <!--   <remap from="cmd_vel_mux/output" to="cmd_vel"/> -->
  <!-- </node> -->

</launch>
